CONTRIBUTING.md

```diff
@@ -66,6 +66,10 @@ Packages are located within the `packages` folder of the repository. Each packag
 
 If you'd like to contribute by fixing a bug, implementing a feature, or even correcting typos in our documentation, you'll want to submit a pull request. Before submitting a pull request, be sure to [rebase](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) your branch from master. Do not use ``git merge`` or the *merge* button provided by GitHub.
 
+### Merging a pull request
+
+If you are merging a pull request, be sure to use the pull request title as the commit title. The title should follow the [conventional commit guidelines](https://www.conventionalcommits.org/). It is recommended that if you are merging in pull requests regularly that you add a browser extension that will auto-correct the title for you.
```

Comment:
Should we link to an extension or two that does this?

Reply:
I considered it but I thought we might wait until we come to a consensus on which one the community likes best. Or I could just link the one I tested. What's the preference?